docs/concepts/index.md

```diff
@@ -1,6 +1,62 @@
+   
+## Features
+
+> See [the doc](xref:65F8FFAC-CD1F-4304-BDFF-423936F0B08A) for a full list of controls with screenshots
+
+- [ARSceneView](xref:D07D974D-8587-428E-BC75-EF531108515B): Part of the AR Toolkit, enables integration of GIS content and ARKit/ARCore.
+- [Bookmarks](xref:B1CCBC8A-91A2-485A-97E0-CF2B9F692B26): Shows bookmarks, from a map, scene, or a list; navigates the associated MapView/SceneView when a bookmark is selected.
+- **Compass**: Shows a compass direction when the map is rotated. Auto-hides when the map points north up.
+- **FeatureDataField**: Displays and optionally allows editing of a single field attribute of a feature.
+- **Legend**: Displays a legend for a single layer in your map (and optionally for its sub layers).
+- **MeasureToolbar**: Allows measurement of distances and areas on the map view.
+- **PopupViewer**: Display details and media, edit attributes, geometry and related records, and manage the attachments of features and graphics (popups are defined in the popup property of features and graphics).
+- **ScaleLine**: Displays current scale reference.
+- **SymbolDisplay**: Renders a symbol in a control.
+- **TimeSlider**: Allows interactively defining a temporal range (i.e. time extent) and animating time moving forward or backward.  Can be used to manipulate the time extent in a MapView or SceneView.
+
+### Features in Preview
+
+- **ChallengeHandler**: Displays SignInForm when accessing secure ArcGIS resources, as well as helper classes for storing credentials in Windows' credentials cache.
+- **SignInForm**: Displays a UI dialog to enter or select credentials to use when accessing secure ArcGIS resources.
+
+## Resources
+
+- [Documentation](https://esri.github.io/arcgis-toolkit-dotnet/)
+- [List of controls](xref:65F8FFAC-CD1F-4304-BDFF-423936F0B08A)
+- [Build the toolkit](xref:3CCE1627-57F9-4ECB-A650-6D4D2E343337)
+- [System Requirements](https://esri.github.io/arcgis-toolkit-dotnet/requirements.html)
+- [ArcGIS Runtime SDK for .NET](http://esriurl/dotnetsdk)
+
+## Issues
+
+Find a bug or want to request a new feature? Please let us know by [submitting an issue](https://github.com/Esri/arcgis-toolkit-dotnet/issues/new).
+
+## Contributing
+
+Anyone and everyone is welcome to [contribute](CONTRIBUTING.md).
+
+## Licensing
+
+Copyright © 2014-2020 Esri.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+A copy of the license is available in the repository's [license.txt](/license.txt) file.
```

Comment:
I don't get why a lot of this is on the index page of the getting started section? (ie issues, contributing, licensing, resources)